types/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
This file is not _required_ unless we want linting capabilities _within_ this repo.